compiler/src/vcbuild/druntime.vcxproj

```diff
@@ -0,0 +1,133 @@
+    <LinkIncremental Condition="'$(Configuration)'=='Release'">false</LinkIncremental>
+    <CCModel Condition="'$(Platform)'=='x64'">x64</CCModel>
+    <CCModel Condition="'$(Platform)'=='Win32'">x86</CCModel>
+    <DMDModel Condition="'$(Platform)'=='x64'">64</DMDModel>
+    <DMDModel Condition="'$(Platform)'=='Win32'">32mscoff</DMDModel>
+    <DMDConfig Condition="$(IsDebug)">Debug</DMDConfig>
+    <DMDConfig Condition="!$(IsDebug)">Release</DMDConfig>
+    <TargetName>druntime$(DMDModel)</TargetName>
+    <DMDBinDir>..\..\..\generated\Windows\$(Configuration)\$(PlatformName)\</DMDBinDir>
+    <MakeTarget>
+    </MakeTarget>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
+      <WarningLevel>Level3</WarningLevel>
+      <RuntimeLibrary Condition="'$(Configuration)'=='Release'">MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary Condition="'$(Configuration)'!='Release'">MultiThreadedDebug</RuntimeLibrary>
+    </ClCompile>
+    <DCompile>
+      <CompilationModel>Project</CompilationModel>
+      <ImportPaths>
+      </ImportPaths>
+      <CRuntimeLibrary Condition="'$(Configuration)'=='Release'">MultiThreaded</CRuntimeLibrary>
+      <CRuntimeLibrary Condition="'$(Configuration)'!='Release'">MultiThreadedDebug</CRuntimeLibrary>
+      <AdditionalOptions>-lib -preview=shortenedMethods -preview=fieldwise
+ -preview=dtorfields -preview=dip1000</AdditionalOptions>
+      <ShowCommandLine>true</ShowCommandLine>
+      <PreviewShortenedMethods Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+      </PreviewShortenedMethods>
+    </DCompile>
+    <Link>
+      <SubSystem>Console</SubSystem>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
+    <ClCompile>
+      <Optimization>MaxSpeed</Optimization>
+      <FunctionLevelLinking>true</FunctionLevelLinking>
+      <IntrinsicFunctions>true</IntrinsicFunctions>
+      <PreprocessorDefinitions>WIN32;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <DCompile>
+      <Optimizer>true</Optimizer>
+      <Inliner>true</Inliner>
+      <DebugCode>Release</DebugCode>
+    </DCompile>
+    <Link>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\vcbuild\dmd.vcxproj">
+      <Project>{c4da4faf-a928-4230-8353-e80c76f223e0}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\..\druntime\src\__importc_builtins.di" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\..\druntime\src\importc.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <Dcompile Include="..\..\..\druntime\src\core\*.d" />
```

Comment:
There is a bad lowercase 'c' here.

Reply:
importc is spelled with lowercase c. Otherwise I have no idea what lowercase c that you are referring to.